easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -237,10 +246,10 @@ public void onClick(DialogInterface dialog, int which) {
                                 Intent intent = new Intent(Settings.ACTION_APPLICATION_DETAILS_SETTINGS);
                                 Uri uri = Uri.fromParts("package", activity.getPackageName(), null);
                                 intent.setData(uri);
-                                activity.startActivity(intent);
+                                startAppSettingsScreen(object, intent);
```

Comment:
I think it is still needed, since we need to retrieve the package name for URI
`Uri uri = Uri.fromParts("package", activity.getPackageName(), null);`